.circleci/config.yml

```diff
@@ -87,10 +100,10 @@ workflows:
             - build-linux-go-1.12
           filters:
             tags:
-              only: /.*/
+              only: /.*/ 
       - release-linux-go-1.12:
           requires:
-            - tests-linux-go-1.12
+            - whiteblock-test
```

Comment:
```suggestion
            - test-whiteplock-go-1.12
```

Reply:
Assuming that for a (linux, et al) release we don't want to replace tests with whiteblock, but to augment them?